docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -461,10 +461,32 @@ Several parameters must be set properly to make sure the flight controller can c
+will begin outputting active throttles where the pulses will never be less than 1000us and will never exceed 2000us.
+
+We highly recommend that you coordinate your PX4 configuration with your module's :ref:`arming parameter settings <manual_advanced_arming>`. For example, suppose your module is 
+configured to disarm on throttle commands between 0 and 2%, and to arm between 2 and 12.5%. When the flight controller is disarmed, we want our module to be as well. Since Vertiq 
+modules use a range of 1000us to 2000us for PWM throttle parsing, we can easily map our percentages to the correct Disarmed, Minimum, and Maximum values. When disarmed, we want the flight 
+controller to output a command that will disarm the module. So, we will set our Disarmed value to 1000us as it represents a 0% throttle. Now, we want to ensure the module arms when 
+the flight controller does, so we'll adjust our minimum flight controller output to fall inside of our :ref:`arming region <arming_throttle_regions>`. To ensure we fall inside of the 
+arming region, we will configure the flight controller to output a 2.5% throttle once armed. 2.5% of our 1000us range is 25, so we'll set our Minimum to 1025us. The maximum is 
+defined by the longest pulse Vertiq modules accept as PWM throttles, 2000us.
+
+.. warning::
+
+  It is strongly encouraged that your Minimum throttle output from the flight controller lies outside of your module's disarming region. If your module disarms while in flight, 
+  it can cause your modules to stop responding to throttle commands until they rearm. For the safest flight experience, your flight controller's outputs and your module's arming 
+  regions must align such that the flight controller will never disarm your module via throttles while the flight controller is armed.
+
+In the module configuration completed so far in this tutorial, there is no configured :ref:`disarming region <disarm_on_throttle>`. So, we will set our Disarmed output to 990us and our 
+Minimum outputs to 1000us in order to ensure that the module will not spin unexpectedly on 0% throttle outputs. Note that our Disarmed output should always be lower 
+than the Minimum output. The Maximum value can remain 2000us.
+
+Once you have fully configured the output channel, you will see sliders appear under Actuator Testing. Please note that if your module is powered on and connected to the MAIN 1 output when it is enabled, 
+you will hear the module play its :ref:`two tone arming song <arming_song>` as PX4 begins transmitting Disarmed throttle commands (990us) immediately on channel activation. By default, Vertiq modules 
 take 0% throttle commands as arming throttles. 
 
 For more information on how to configure the module to properly interpret throttle commands, see the :ref:`manual_throttle` section of the Feature Reference 
```

Comment:
This is for further down: Should we update any of the DSHOT pictures as well to match the PWM stuff?

Reply:
I don't think we need to, honestly. The DSHOT setup is independent of the PWM stuff, so I don't particularly care what's listed in the PWM boxes in the background